man/convert.Rd

```diff
@@ -0,0 +1,208 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/convertFromBIOM.R, R/convertFromDADA2.R,
+%   R/convertFromPhyloseq.R, R/convertToPhyloseq.R
+\name{convert}
+\alias{convert}
+\alias{importBIOM}
```

Comment:
importBIOM is in wrong place.

Otherwise this PR looks good

Reply:
Should I make a new man page for importBIOM only?

Reply:
Do we have other importers in same man page? If there are general man page for importers, add it there. Otherwise, create new one for importBIOM

Reply:
Ok! If that is the case, can you check what is the best approach (should functions have own man page or can they be merged into one bigger man page).

The initial idea was to highlight that these functions work similarly (for example agglomerate* functions). However, merging the man pages into one, have some disadvantages.

Also as we implemented the renaming, the synergy between functions should be evident

Reply:
The main disadvantage I see is that many parameters under the current importer man pages have the same names (e.g. `file`) and I think we would need to describe the use of `file` parameter for all importers in one R file only.

Reply:
Alright, thanks. The documentation fixing was still a thing to do, so let's take this into account.